shared-libs/cht-script-api/test/index.spec.js

```diff
@@ -10,22 +16,23 @@ describe('CHT Script API - index', () => {
   });
 
   it('should call auth.hasPermissions', () => {
+    const authHasPermissions = sinon.stub(auth, 'hasPermissions').returns(true);
     const userRoles = [ 'chw' ];
     const chtPermissionsSettings = {
       can_backup_facilities: [ 'chw', 'national_admin' ],
       can_export_messages: [ 'national_admin', 'chw', 'analytics' ]
     };
+    const permissions = [ 'can_backup_facilities', 'can_export_messages' ];
 
-    const result = chtScriptApi.v1.hasPermissions(
-      [ 'can_backup_facilities', 'can_export_messages' ],
-      userRoles,
-      chtPermissionsSettings
-    );
+    const result = chtScriptApi.v1.hasPermissions(permissions, userRoles, chtPermissionsSettings);
 
     expect(result).to.be.true;
+    expect(authHasPermissions.callCount).to.equal(1);
+    expect(authHasPermissions.args[0]).to.have.members([ permissions, userRoles, chtPermissionsSettings ]);
```

Comment:
Please use `to.deep.equal` here as well.

Reply:
Done!